tools/webpack-make

```diff
@@ -190,7 +190,7 @@ function generateDeps(makefile, stats) {
     lines.push("\trm -rf $(" + prefix + "_OUTPUTS) $(" + prefix + "_INSTALL) " + makefile);
     lines.push("all-local:: " + prefix);
     lines.push("\t@true");
-    lines.push("clean-local:: clean-" + prefix);
+    lines.push("distclean-local:: clean-" + prefix);
```

Comment:
Hmmm, I'd be against breaking the common semantics of ```make clean``` for developers: "clean out the things I just built". There are far more developers than packagers.